pkg/cmd/admin/network/project_options.go

```diff
@@ -84,7 +84,14 @@ func (p *ProjectOptions) Validate() error {
 		errList = append(errList, errors.New("must provide --selector=<project_selector> or projects"))
 	}
 
-	// TODO: Validate if the openshift master is running with mutitenant network plugin
+	clusterNetwork, err := p.Oclient.ClusterNetwork().Get(sdnapi.ClusterNetworkDefault)
+	if err != nil {
+		errList = append(errList, errors.New("could not manage pod network: error on getting the running network plugin"))
+	} else if !sdnapi.IsOpenShiftMultitenantNetworkPlugin(clusterNetwork.PluginName) {
+		err := fmt.Errorf("could not manage pod network: the running plugin %s is not OpenShift mutitenant network plugin", clusterNetwork.PluginName)
```

Comment:
'Using plugin: %q, managing pod network is only supported for openshift multitenant network plugin'?